src/App.js

```diff
@@ -0,0 +1,24 @@
+import "./styles.css";
+import React, { useState } from "react";
+import { Header } from "./Components/header/Header";
+import { Navigation } from "./Components/navigation/Navigation";
+import { Avatar } from "./Components/avatar/Avatar";
+import { Home } from "./Components/home/Home";
+import { Button } from "./Components/button/Button";
+
+export default function App() {
+  const [a, setA] = useState(<Home />);
+
+  return (
+    <div className="App-grid">
+      <Header />
+      <Navigation
+        parentCallback={(b) => {
+          console.log("exp");
+          setA(b);
+        }}
+      />
+      {a}
+    </div>
+  );
```

Comment:
You can remove the console function now. Instead of using "a and b" variable names you can use some meaningful name.

Reply:
Yes, I will remove console.log and replace a, b with meaningful names. Thanks for the suggestion.